magento/magento2
# 2.3 develop : add option exclude to cron:run command to exclude specific group ids

Add the option "exclude" to the command "cron:run" to exclude specific cron group ids from cron:run process.

### Description
The change would add the option exclude to exclude specific cron groups from running, if the command "cron:run" is run.
This will help to further specify, which cron processes are to be executed while other critical processes are run on the server. The option "group" seems to be fine, but could lead to other processes (e.g. of a new extension) to be forgotten. So it would be much safer to exclude a cron group and all others to be run instead of manually addressing all current cron groups except the one, which is meant to be avoided.

### Manual testing scenarios
1. Run the command 'bin/magento cron:run --exclude index' and check, if index crons, which have been scheduled before the exectution of the command were run.
2. Repeat step one with a comma separated string like 'bin/magento cron:run --exclude index, default' oder 'bin/magento cron:run --exclude "index, default"'.
